msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -108,29 +114,21 @@ microsoftTeams.meeting.shareAppContentToStage((err, result) => {
 }, appContentUrl);
 ```
 
-### Response codes
+**Query parameter**
 
-The following table provides the response codes:
+The following table includes the query parameters:
 
-|Response code|Description|
-|---|---|
-| **500** | Internal error. |
-| **501** | API isn't supported in the current context.|
-| **1000** | App doesn't have proper permissions to allow share to stage.|
+|Value|Type|Required|Description|
+|---|---|----|---|
+|`callback`| String | Yes | Callback contains two parameters, error and result. The *error* can contain either an error of type *SdkError* or null when share is successful. The *result* can contain either a true value if there's a successful share or null when the share fails. |
+|`appContentURL`| String | Yes | The URL that is shared on to the stage. |
+| `sharingProtocol` | String | No | *collaborative* (default) or *screenShare* |
```

Comment:
Could we have the same last two rows as the table below (line 282 and 283) instead of line 125 please?

Reply:
updated

  | `shareOptions`| Object | No | Defines additional sharing options.|
  |shareOptions.sharingProtocol | Enum | No | The screen sharing protocol. The supported values are `Collaborative` and `ScreenShare`. Default is `Collaborative`.|